### ISSUE TITLE
migrate all 'null' values to null and migrate the dsd tables

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#1031]
the issue #1030 itself is fixed and tested. unfortunately a new task arises from it: A migration script would have to change all the 'Null' values to null and I have to add a migration to ALL tables in the dsd that use the old 'Null' default.

_Originally posted by @urapadmin in https://github.com/urapadmin/kiosk/issues/1030#issuecomment-822655547_

**Comment by urapadmin** [comment]
this is done so far only for locus because of #961

**Comment by urapadmin** [comment]
done now for all of them. A new bug in migration surfaced during the process: default('Null') and default(Null) where the same. All the tests before only worked because I used default('none'). And so the capitalization made the difference. 

  * Table survey_unit_data had a default('Null') and I tried to change it accidentally to default(Null) and that failed because 'Null' and Null came out both as Null. Now there is a difference between null and 'null: [https://www.meritaten.net/urapdev/doku.php?id=manuals:developer_manual:dsd_syntax#null_values](https://www.meritaten.net/urapdev/doku.php?id=manuals:developer_manual:dsd_syntax#null_values)
 
now all tables are migrated to default(null). I also changed the few that used default("Null"), so using double quotes.
